app/code/Codifi/Customer/view/adminhtml/ui_component/customer_listing.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<listing xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Ui:etc/ui_configuration.xsd">
+    <columns name="customer_columns" class="Magento\Customer\Ui\Component\Listing\Columns">
+        <column name="taxvat" sortOrder="180">
+            <argument name="data" xsi:type="array">
+                <item name="config" xsi:type="array">
+                    <item name="componentDisabled" xsi:type="boolean">true</item>
+                </item>
+            </argument>
+            <settings>
```

Comment:
You don't need to copy all nodes
`The instance configuration file name is the name of instance (%instance_name%). The namespace of the names is global; meaning that if the file names in different modules are the same, they are merged into a single configuration for the particular instance.`
https://devdocs.magento.com/guides/v2.4/ui_comp_guide/concepts/ui_comp_xmldeclaration_concept.html#about-the-basic-component-configuration-file